File: basic varialbe assignment.cpp
/*
This code should store "codewa.rs" as a variable called name but it's not working. Can you figure out why?

***** BUG *****
#include <string>

std::string a == "code";
std::string b == "wa.rs";
std::string name == a + b;

*/

#include <string>

std::string a = "code";
std::string b = "wa.rs";
std::string name = a + b;

File: basic varialbe assignment.js
/*
This code should store "codewa.rs" as a variable called name but it's not working. Can you figure out why?

***** BUG *****
var a == "code";
var b == "wa.rs";
var name == a + b;

*/

const a = "code";
const b = "wa.rs";
let name = `${a}${b}`;